bundles/best_practices.rst

```diff
@@ -116,23 +116,20 @@ files are going to be part of the repository.
 The following classes and files have specific emplacements (some are mandatory
 and others are just conventions followed by most developers):
 
-===============================  =============================  ================
-Type                             Directory                      Mandatory?
-===============================  =============================  ================
-Commands                         ``Command/``                   Yes
-Controllers                      ``Controller/``                No
-Service Container Extensions     ``DependencyInjection/``       Yes
-Event Listeners                  ``EventListener/``             No
-Model classes [1]                ``Model/``                     No
-Configuration                    ``Resources/config/``          No
-Web Resources (CSS, JS, images)  ``Resources/public/``          Yes
-Translation files                ``Resources/translations/``    Yes
-Templates                        ``Resources/views/``           Yes
-Unit and Functional Tests        ``Tests/``                     No
-===============================  =============================  ================
-
-[1] See :doc:`/doctrine/mapping_model_classes` for how to handle the
-mapping with a compiler pass.
+===============================  ========================================  ==========
+Type                             Directory                                 Mandatory?
+===============================  ========================================  ==========
+Commands                         ``Command/``                              Yes
+Controllers                      ``Controller/``                           No
+Service Container Extensions     ``DependencyInjection/``                  Yes
+Doctrine entities and documents  ``Entity/`` (ORM) or ``Document/`` (ODM)  No 
```

Comment:
Because thanks to the `doctrine.orm.mapings` you can store your entities anywhere you like, even outside the bundles.

In any case, I dislike this `Mandatory` column more and more. The intention is good, but this cannot simply be answered by Yes or No. I propose to remove this column and add a new article explaining how to override the location every single part of the bundle: controllers, templates, entities, commands, etc.